docs/src/visualization/narratives.md

```diff
@@ -1,14 +1,4 @@
 # Nextstrain Narratives  
-<!-- WARNING -->
```

Comment:
This page includes links to targets like `README.md#Help` that don't get rendered as expected by RTD. We should do a quick 404 link check on the rendered site to find all of these broken links and update their targets accordingly.